packages/html-reporter/src/reportContext.tsx

```diff
@@ -0,0 +1,29 @@
+/*
+  Copyright (c) Microsoft Corporation.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+import * as React from 'react';
+import type { HTMLReport } from './types';
+
+
+export const HTMLReportContext = React.createContext<HTMLReport | undefined>(undefined);
```

Comment:
```suggestion
const HTMLReportContext = React.createContext<HTMLReport | undefined>(undefined);
```

you already have the hook as a public interface, no need to expose the context itself